### DISCUSSION TITLE
Completely stumped - tail not tailing log file, not sending any lines

### DISCUSSION THREAD

**Discussion by drewmiranda-gl** [fluent/fluentd#4261] [Q&A]
Quick intro, i'm new to using fluentd and wanted to do some quick testing. I naively thought it would be easy but i'm completely stumped. I have a strong background in IT and programming, so i'm familiar with basic concepts of editing text files, installing things on linux, networking, client service architecture, the works.

Where i'm stuck is trying to get fluentd to do anything with a a text file source (`@type tail`). I'm read the docs and looked at and tried many example but nothing works. 

Where i'm at:

* Ubuntu 22
* Fluentd installed

config:

```xml
<source>
  @type sample
  sample {"hello":"world"}
  tag test
</source>

<source>
  @type tail
  path /home/drew/*.log
  read_from_head true
  tag test
  <parse>
    @type none
    message_key test
  </parse>
</source>

<match test>
  @type copy
  <store>
    @type file
    path /var/log/td-agent/testlogfolder
  </store>
</match>
```

The above works but ONLY for the sample source, meaning i see the output in the match log path 
`/var/log/td-agent/testlogfolder/buffer.b601f5f4097182fd71addff04c17b07c5.log`

I should note that even enabling verbose logging `-vv` does not show any errors or any clues or anything. I've also tried `pos_file` but it didn't help nor change anything. complete mystery.

no matter what i try, fluentd appears to be ignoring the tail source. am i missing something?

**Comment by daipom** [comment]
Could you please share the version of Fluentd?
If you use `td-agent`, you can take it by `$ td-agent --version`.

**Comment by drewmiranda-gl** [comment]
Thanks for the reply!

```
drew@placeholder:~$ td-agent --version
td-agent 4.5.0 fluentd 1.16.1 (0a6d706a9cee5882d751b2cc6169696709df0134)
```

**Comment by daipom** [comment]
> The above works but ONLY for the sample source, meaning i see the output in the match log path 
`/var/log/td-agent/testlogfolder/buffer.b601f5f4097182fd71addff04c17b07c5.log`

I'm not sure what this means.
Don't the logs in `/home/drew/*.log` exist in the files being added into  `/var/log/td-agent/testlogfolder/`?

**Reply by drewmiranda-gl** [reply]
Logs do exist in `/home/drew/`, i was testing with a file called `/home/drew/log.log` (and had that configured as the source but in the config i copied it has the wildcard i tried for testing). Using the log file to test the behavior of td-agent. Using the output in `/var/log/td-agent/testlogfolder/` to test what the output looks like and that td-agent is collecting logs from the source.

As far as i can tell and following the reference documentation this is how someone would add a log file to be collected.
